### PR TITLE
Add Integtest.sh for OpenSearch integtest setups

### PR DESCRIPTION
**Pull request by peterzhuamazon**
Signed-off-by: Peter Zhu <zhujiaxi@amazon.com>

### Description
Add Integtest.sh for OpenSearch integtest setups

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: integtest.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  PASSWORD=`echo $CREDENTIAL | awk -F ':' '{print $2}'`
+fi
+
+if [ -z "$DIRECTORY" ]
+then
+    DIRECTORY="root"
+fi
+
+
+if [ "$DIRECTORY" = "dashboards-notebooks" ]
```
Comment (joshuali925) [inline integtest.sh:86]: this should be dashboards-reports?

**Review by joshuali925 — APPROVED** [review]
one comment, thanks

File: integtest.sh (hunk 1 above)
Comment (peterzhuamazon) [reply integtest.sh:86]: typo

**Review by davidcui-amzn — DISMISSED** [review]
Please change directory check to `dashboards-reports` as per Josh's comment- other than that, LGTM

**Review by joshuali925 — APPROVED** [review]

**Review by davidcui-amzn — DISMISSED** [review]

**Review by zhongnansu — APPROVED** [review]

**Review by joshuali925 — APPROVED** [review]

**Review by davidcui-amzn — APPROVED** [review]

**Pull request merged**